### PR TITLE
add(permission sets): add three new permission sets

### PR DESCRIPTION
**Pull request by crlptl**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: identity-and-access-management/iam/reference-content/permission-sets.mdx [hunk 1]
```diff
@@ -97,6 +97,9 @@ Below is a list of the permission sets available at Scaleway.
 | SecretManagerReadOnly        | List and read secrets' metadata (name, tags, creation date, etc.). Does not include permissions for data (versions) accessing or editing                            |
 | SecretManagerFullAccess      | Full access to create, read, list, edit and delete secrets in Secret Manager                |
 | SecretManagerSecretAccess    | Full access to secrets' metadata and data (versions) in Secret Manager. Does not include permissions for data editing             |
+| SecretManagerSecretCreate    | Create permission to secret manager secrets and their versions. Does not include update permission on secrets, versions and folders             |
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/permission-sets.mdx:100]:
```suggestion
| SecretManagerSecretCreate    | Create permission for secret manager secrets and their versions. Does not include update permission on secrets, versions and folders             |
```

File: identity-and-access-management/iam/reference-content/permission-sets.mdx [hunk 2]
```diff
@@ -97,6 +97,9 @@ Below is a list of the permission sets available at Scaleway.
...
+| SecretManagerSecretDelete    | Delete permission to secret manager secrets and their versions             |
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/permission-sets.mdx:101]:
```suggestion
| SecretManagerSecretDelete    | Delete permission for secret manager secrets and their versions             |
```

File: identity-and-access-management/iam/reference-content/permission-sets.mdx [hunk 3]
```diff
@@ -97,6 +97,9 @@ Below is a list of the permission sets available at Scaleway.
...
+| SecretManagerSecretWrite     | Write permission to secret manager secrets and their versions. Does not include creation permission on secrets, versions and folders             |
```
Comment (RoRoJ) [inline identity-and-access-management/iam/reference-content/permission-sets.mdx:102]:
```suggestion
| SecretManagerSecretWrite     | Write permission for secret manager secrets and their versions. Does not include creation permission on secrets, versions and folders             |
```

**Review by RoRoJ — COMMENTED** [review]
"For" seems more right to me, but totally willing to discuss if you think otherwise!

File: identity-and-access-management/iam/reference-content/permission-sets.mdx (hunk 1 above)
Comment (nerda-codes) [inline identity-and-access-management/iam/reference-content/permission-sets.mdx:100]:
```suggestion
| SecretManagerSecretCreate    | Permission to create secrets and their versions in Secret Manager. Does not include permission to update secrets, versions and folders             |
```

File: identity-and-access-management/iam/reference-content/permission-sets.mdx (hunk 2 above)
Comment (nerda-codes) [inline identity-and-access-management/iam/reference-content/permission-sets.mdx:101]:
```suggestion
| SecretManagerSecretDelete    | Permission to delete secrets and their versions  in Secret Manager            |
```

**Review by nerda-codes — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**